replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/NodeAdapters.java

```diff
@@ -0,0 +1,26 @@
+package org.codice.ditto.replication.api.impl;
+
+import java.util.List;
+import org.codice.ditto.replication.api.NodeAdapterFactory;
+import org.codice.ditto.replication.api.NodeAdapterType;
+
+public class NodeAdapters {
```

Comment:
❗️ Javadoc. This comments applies to a lot of places. All classes and public methods should always be javadoc